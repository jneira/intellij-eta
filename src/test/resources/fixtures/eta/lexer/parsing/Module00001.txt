ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Module00001')
WHITE_SPACE (' ')
IToparen ('(')
ITcparen (')')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITlineComment ('-- Parse an empty export list.')
WHITE_SPACE ('\n')
ITvocurly ('')