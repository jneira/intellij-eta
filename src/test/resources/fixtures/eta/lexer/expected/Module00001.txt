ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Module0000')
WHITE_SPACE ('1')
IToparen (' ')
WHITE_SPACE ('')
ITcparen ('(')
WHITE_SPACE (')')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITlineComment ('-- Parse an empty export list.')
WHITE_SPACE ('\n')
ITvocurly ('')